mtags/src/main/scala-3/scala/meta/internal/pc/SemanticdbSymbols.scala

```diff
@@ -48,7 +48,20 @@ object SemanticdbSymbols:
                   //   however in scalac this method is defined only in `module Files`
                   if typeSym.is(JavaDefined) then
                     typeSym :: owner.info.decl(termName(value)).symbol :: Nil
+                  /**
+                   * Looks like decl doesn't work for:
+                   *  package a:
+                   *   implicit class A (i: Int):
+                   *      def inc = i + 1
+                   */
+                  else if typeSym == NoSymbol then
+                    val searched = typeName(value)
+                    owner.info.allMembers
+                      .find(_.name == searched)
+                      .map(_.symbol)
+                      .toList
```

Comment:
Wouldn't `owner.info.member(termName(value)).symbol` achieve the same?

Also can you in the comment mark for which part it doesn't work? (I assume you mean `implicit class A`, since that's the type.)

Reply:
I agree, but maybe with `typeName` instead of `termName`?

Reply:
Thanks, changed and it works!